src/main/java/com/minecolonies/coremod/client/gui/WindowHireWorker.java

```diff
@@ -289,6 +289,7 @@ else if ((building.getWorkerId().size() >= building.getMaxInhabitants()) && !bui
                 final TextBuilder textBuilder = PaneBuilders.textBuilder();
                 int skillCount = citizen.getCitizenSkillHandler().getSkills().entrySet().size();
 
+                textBuilder.append(new StringTextComponent(""));
```

Comment:
Yep, I think it's vanilla code :D but gonna do a bit more deep search

Reply:
There is a lot of black magic in text processing for me, trying to debug it for half an hour and without proper naming it's literally impossible. Since the style pollution bug only happens if the first skill is styled, I blame vanilla component appending and line split mechanism for accidentally polluting all siblings with empty style

Reply:
Okey